guide/how-it-works/ecash/design-best-practices.md

```diff
@@ -0,0 +1,496 @@
+
+3. **Quick Mint Access**: Provide easy access to mint or federation details directly from the mint or federation list. This allows users to interact with mints or federations more fluidly without needing to navigate through multiple screens.
+
+#### Pending tokens
+{% include image-gallery.html pages = page.images_pending %}
+
+It's useful to display generated tokens in the transaction history along with their status—whether pending or redeemed. This helps users quickly assess their transaction status. Providing the ability to expand on a token to view details and take actions like resending or redeeming is also important. This feature becomes particularly valuable if a token was lost or never claimed.
+
+## Cashu best practices
+
+### Default Mint
+{% include image-gallery.html pages = page.images_mint-default %}
+When designing for Cashu, consider allowing users to set a default mint. Since Cashu mints are independently operated, users may prefer to use a trusted mint for their transactions. By allowing them to select a default mint, you simplify their experience and reduce the risk of exposure to a potentially malicious mint. Additionally, consider offering auto-swap preferences.
+
+### Auto-Swapping to a Default Cashu Mint
+{% include image-gallery.html pages = page.images_auto-swap %}
+Wallets can provide the option to automatically swap ecash from an unknown mint to their a default mint. This optional feature can simplify transactions by routing all ecash transactions through the assigned default mint. An auto-swap requires a lightning payment..
+
+{% include tip/open.html color="blue" icon="info" label="Auto-Swapping Involves Fees" %}
+
+It's important to inform users that auto-swapping involves fees and that the amount depends on the amount of ecash they want to swap and the current network conditions.
+
+{% include tip/close.html %}
+
+
+### Backup and restore for cashu
+The Cashu backup and restoration process is designed to ensure users can securely recover their wallets and maintain access to their bitcoin backed ecash tokens in the event they switch devices or experience data loss. **Users should only use their recovery seed phrase once.** Repeated use of the seed phrase for restoration can lead to synchronization issues and potential errors. This is due to the fact that each time you restore, you might be dealing with an outdated state of the wallet, which can cause discrepancies in token balances and transactions.
+
+{% include tip/open.html color="blue" icon="info" label="Single Use Recovery" %}
+
+After successfully restoring your wallet using a recovery seed phrase, it is highly recommended that you generate a new recovery seed phrase immediately. This step is crucial because the original recovery phrase has now been used and could be more susceptible to synchronization risks.
+
```

Comment:
Sync issues would happen only if the seed is being used in two or more places. If its not being used in the old place anymore it would be okay. Though if the seed is potentially compromised it should not be used anymore.

Reply:
So, if I understand correctly, reusing the seed phrase more than once is fine as long as there isn’t another active instance of the wallet elsewhere. For example:

A sync issue **won’t** occur when:
1. The user factory resets their wallet and restores it using the seed phrase.
3. Two weeks later, the user factory resets their wallet again.
4. The user restores their wallet using the same seed phrase.

Reply:
right

Reply:
As long as they don't have two instances of their wallet then using the same seed phrase should be okay. I'll update this when I can.